### PR TITLE
updated README, still needs work

### PR DESCRIPTION
**Pull request by earroyo28**
Definitely still needs work, I'm still working on figures for the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# main-project
\ No newline at end of file
+# Project TBD [Tau Be Damned]
+
+-------------------------
+
+The project TBD [Tau Be Damned] aims to use the amino acide sequence of a protein to identify whether it is disordered.
```
Comment (jiaweih) [inline README.md:5]: acid?

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+## Project Objective
+
+-------------------------
+
+Our goal is to build a tool to identigy whether a protein is disordered bases on its amino acid sequence. We have collected amino acid
```
Comment (jiaweih) [inline README.md:11]: "identify"; "based on"

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+sequences for ordered and disordered proteins from publicly available datasets to train a machine learning model to perform the
+identificaction task.
```
Comment (jiaweih) [inline README.md:13]: classification task

File: README.md [hunk 4]
```diff
@@ -1 +1,51 @@
-# main-project
\ No newline at end of file
+# Project TBD [Tau Be Damned]
```
Comment (jiaweih) [inline README.md:1]: It's a good start

**Comment by rjpecoraro** [comment]
Now that I've merged this, should I delete the branch?

**Comment by jiaweih** [comment]
To delete the branch is fine. By the way, the issue was not addressed, why is it merged?

**Comment by rjpecoraro** [comment]
You mean the typos? I figured they were easy enough to fix and having a starting file is necessary for me to add some of my own things to it. I just created a separate branch with a pull request though if you want to approve the edits.

**Pull request merged**